### ISSUE TITLE
Adding spring cloud eureka to sample app causes test failure

### ISSUE THREAD

**Issue by sameer-pandit** [bclozel/spring-boot-web-reactive#37]
I tried adding spring cloud eureka, to pom.xml (for reactor-netty profile)of spring-boot-sample-web-reactive, I am getting the below error in test of sample. What is the correct way to use eureka discovery service for spring boot web reactive app.

> Caused by: org.springframework.context.ApplicationContextException: Unable to start embedded container; nested exception is org.springframework.context.ApplicationContextException: Unable to start ReactiveWebApplicationContext due to missing ReactiveHttpServerFactory bean.

**Comment by bclozel** [comment]
This starter is still experimental and should be merged (in some form) for Spring Boot 2.0.
Note that not all Spring projects support the new reactive web project nor Spring 5 at the moment, so this needs to be addressed in the dedicated project first.

TL;DR this is not supported for now.
Feel free to browse existing issues or create a new one in spring-cloud-eureka about this.

Thanks for trying this starter!

**Comment by sameer-pandit** [comment]
Thanks for quick reply , As this [page](http://docs.spring.io/spring-framework/docs/5.0.0.M1/spring-framework-reference/html/web-reactive.html) points that we can use spring web reactive with manual bootstrapping. Can you point to some code sample which has this way of creating spring app? I am not much experienced with  Spring but want to get hands dirty on reactive stuff (which only Spring seems done a lot of work in this area)

**Comment by bclozel** [comment]
This starter is by far the easiest way to get started with the new web reactive framework.
You can of course bootstrap things manually, as described on the page you mentioned.

It seems you've already tried this since you created this issue with a fairly advanced use case.
